src/components/v2/V2Project/V2PayButton/V2ConfirmPayModal/index.tsx

```diff
@@ -23,8 +26,33 @@ import { weightedAmount } from 'utils/v2/math'
 import TransactionModal from 'components/TransactionModal'
 import Callout from 'components/Callout'
 import useMobile from 'hooks/Mobile'
+import { maxEligibleRewardTier } from 'utils/v2/nftRewards'
+import { featureFlagEnabled } from 'utils/featureFlags'
 
 import { V2PayForm, V2PayFormType } from '../V2PayForm'
+import { NftReward } from './NftRewardCell'
+
+const MOCK_NFTs: NFTRewardTier[] = [
```

Comment:
Nice approach. Maybe we can put this in a separate file for now?

Reply:
Done